02LessonTwo/homework/devon norvell bio

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
Make sure this is a .html file! Change the file name to devon-norvell-bio.html, or it won't render correctly